qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/jupyterhub/main.tf

```diff
@@ -83,6 +83,8 @@ resource "helm_release" "jupyterhub" {
             name = service
             admin = true
             api_token = random_password.service_token[service].result
+            # come back and clean up
+            url = "http://kbatch-kbatch-proxy.${var.external-url}.svc.cluster.local"
```

Comment:
Shouldn't this be `url = "http://kbatch-kbatch-proxy.${var.namespace}.svc.cluster.local"`?

Reply:
Good catch! It should be namespace. Ultimately, I'm not sure we even need this `url`. I believe this is what is used to populate this drop down:
![Screen Shot 2022-04-27 at 15 28 00](https://user-images.githubusercontent.com/42120229/165641828-23e3054f-1687-4377-b68d-81971026f46c.png)
Which at this point doesn't lead you anywhere interesting:
![Screen Shot 2022-04-27 at 15 27 47](https://user-images.githubusercontent.com/42120229/165641867-53ceb505-b8dc-43e7-aee6-9890053b2d12.png)

I think I will remove `url` for now, unless you think we should keep it.